options.go

```diff
@@ -40,8 +40,9 @@ type Options struct {
 
 // RequestOptions are request related options
 type RequestOptions struct {
-	Encoding     encoding.Encoding `short:"e" long:"encoding" description:"The encoding of the data, options are: Thrift, JSON, raw. Defaults to Thrift if the method contains '::' or a Thrift file is specified"`
+	Encoding     encoding.Encoding `short:"e" long:"encoding" description:"The encoding of the data, options are: Thrift, proto, JSON, raw. Defaults to Thrift if the method contains '::' or a Thrift file is specified"`
 	ThriftFile   string            `short:"t" long:"thrift" description:"Path of the .thrift file"`
+	ProtoSet     []string          `long:"protoset" description:"A binary file containing a compiled protobuf FileDescriptorSet."`
```

Comment:
I know grpcurl uses `--protoset` but I'm not crazy about it, "Proto Set" isn't really a Protobuf concept. What about `--file-descriptor-set-file` or `--file-descriptor-set-bin`? And add a shorthand as well.

Reply:
Went with `file-descriptor-set-bin` added `-F` as short. There is already a lowercase `f`, so I'm not a big fan but it already seems like there are more cases where upper and lowercase short are for completely different arguments.